Dockerfile

```diff
@@ -7,16 +7,14 @@ RUN npm install
 COPY . .
 RUN npm run build
 
-COPY ./src/examples/poi-crosschecker/proto/NPOIMessage.proto ./dist/src/examples/poi-crosschecker/proto/
 WORKDIR ./dist/src/examples/poi-crosschecker
 
 ENV GRAPH_NODE_HOST "host.docker.internal"
 ENV ETH_NODE "host.docker.internal:8545"
 ENV NETWORK_URL "https://gateway.testnet.thegraph.com/network"
 ENV TERM "xterm-256color"
-ENV GOERLI_NODE "<GOERLI_NODE>"
 ENV REGISTRY_SUBGRAPH "https://api.thegraph.com/subgraphs/name/juanmardefago/gossip-network-subgraph"
-ENV INDEXER_MANAGEMENT_SERVER "host.docker.internal:18000"
-ENV RADIO_OPERATOR_PRIVATE_KEY "<RADIO_OPERATOR_PRIVATE_KEY>"
+ENV INDEXER_MANAGERMENT_SERVER "host.docker.internal:18000"
+ENV RADIO_OPERATOR_PRIVATE_KEY 806ccd0301ec455bfc94c71c9d199de4ce8838717a15da0a57a5d2cdc717bb5c
```

Comment:
should we hardcode this 👀

Reply:
lol definitely not, thanks for noticing!